### PR TITLE
Update Midstack Dependencies

### PR DESCRIPTION
**Pull request by rusheb**
When trying to build this project I received the following error
```
Execution failed for task ':contracts:compileKotlin'.
> Could not resolve all files for configuration ':contracts:compileClasspath'.
   > Could not resolve com.r3.corda.lib.tokens:tokens-contracts:1.1-RC05.
     Required by:
         project :contracts
      > Could not resolve com.r3.corda.lib.tokens:tokens-contracts:1.1-RC05.
         > Could not get resource 'https://ci-artifactory.corda.r3cev.com/artifactory/corda-lib-dev/com/r3/corda/lib/tokens/tokens-contracts/1.1-RC05/tokens-contracts-1.1-RC05.pom'.
            > Could not GET 'https://ci-artifactory.corda.r3cev.com/artifactory/corda-lib-dev/com/r3/corda/lib/tokens/tokens-contracts/1.1-RC05/tokens-contracts-1.1-RC05.pom'. Received status code 409 from server: Conflict
```

I think this is resolved by this the first commit, which bumps the tokens version from `1.1-RC05` to `1.1`

### CODE REVIEW

**Comment by rusheb** [comment]
Update: I also had to add the confidential identities dependencies in order for the following test to pass:

```kotlin
class FlowTests {
    private val network = MockNetwork(
        MockNetworkParameters()
            .withCordappsForAllNodes(
                listOf(
                    TestCordapp.findCordapp("com.r3.corda.lib.accounts.contracts"),
                    TestCordapp.findCordapp("com.r3.corda.lib.accounts.workflows")
                )
            )
    )
    val alice = network.createNode()
    val bob = network.createNode()

    @Before
    fun setup() = network.runNetwork()

    @After
    fun tearDown() = network.stopNodes()

    private fun createAccount(host: StartedMockNode, name: String): StateAndRef<AccountInfo> {
        val future = host.startFlow(CreateAccount(name))
        network.runNetwork()
        return future.get()
    }

    private fun requestNewKey(host: StartedMockNode, forWhom: AccountInfo): AnonymousParty {
        val future = host.startFlow(RequestKeyForAccount(forWhom))
        network.runNetwork()
        return future.get()
    }

    @Test
    fun `can request key from non-host`() {
        val dan = createAccount(alice, "dan").state.data
        val danAnonParty = requestNewKey(bob, dan)
    }
}
```

Without the Confidential Identities dependency, the following error is thrown: 
```java.lang.NoClassDefFoundError: com/r3/corda/lib/ci/workflows/RequestKeyFlow```

**Comment by alch3mistdev** [comment]
@rusheb , in addition to the buildscript.ext changes, could you also add 

```cordapp "$confidential_id_release_group:ci-workflows:$confidential_id_release_version"```

to the workflows build.gradle and change the FlowTests scaffolding to:

```
private val network = MockNetwork(
        MockNetworkParameters()
            .withCordappsForAllNodes(
                listOf(
                    TestCordapp.findCordapp("com.r3.corda.lib.accounts.contracts"),
                    TestCordapp.findCordapp("com.r3.corda.lib.accounts.workflows")
                )
            )
    )
    val alice = network.createNode()
    val bob = network.createNode()

    @Before
    fun setup() = network.runNetwork()

    @After
    fun tearDown() = network.stopNodes()

    @Test
    fun `dummy test`() {}
}
```

File: build.gradle [hunk 1]
```diff
@@ -3,9 +3,11 @@ buildscript {
         corda_release_group = 'net.corda'
         corda_release_version = '4.3'
         tokens_release_group = 'com.r3.corda.lib.tokens'
-        tokens_release_version = '1.1-RC05'
+        tokens_release_version = '1.1'
```
Comment (rogersanick) [inline build.gradle]: We should jump straight to the latest public version of the SDK. Right now I believe that's `1.2`

File: workflows/build.gradle [hunk 2]
```diff
@@ -59,7 +59,9 @@ dependencies {
     // Token SDK dependencies.
     cordaCompile "$tokens_release_group:tokens-money:$tokens_release_version"
     cordaCompile "$tokens_release_group:tokens-workflows:$tokens_release_version"
+    cordapp "$tokens_release_group:tokens-selection:$tokens_release_version"
```
Comment (rogersanick) [inline workflows/build.gradle]: In the Tokens SDK 1.2, `money` is now part of workflows

File: workflows/src/test/kotlin/com/template/FlowTests.kt [hunk 3]
```diff
@@ -1,14 +1,29 @@
...
+    private val network = MockNetwork(
+        MockNetworkParameters()
+            .withCordappsForAllNodes(
+                listOf(
+                    TestCordapp.findCordapp("com.r3.corda.lib.accounts.contracts"),
+                    TestCordapp.findCordapp("com.r3.corda.lib.accounts.workflows"),
+                    TestCordapp.findCordapp("com.r3.corda.lib.tokens.contracts"),
+                    TestCordapp.findCordapp("com.r3.corda.lib.tokens.workflows"),
+                    TestCordapp.findCordapp("com.r3.corda.lib.tokens.money"),
+                    TestCordapp.findCordapp("com.r3.corda.lib.tokens.selection"),
```
Comment (rogersanick) [inline workflows/src/test/kotlin/com/template/FlowTests.kt]: I believe you don't need to explicitly list `selection` here:

See the JIT MockNetwork tests found [here](https://github.com/corda/token-sdk/blob/1.2/workflows/src/test/kotlin/com/r3/corda/lib/tokens/workflows/JITMockNetworkTests.kt): 

                    TestCordapp.findCordapp("com.r3.corda.lib.tokens.money"),
                    TestCordapp.findCordapp("com.r3.corda.lib.tokens.contracts"),
                    TestCordapp.findCordapp("com.r3.corda.lib.tokens.workflows")

**Review by rogersanick — CHANGES_REQUESTED** [review]
Hey @rusheb, couple of minor changes! Thank you very much for putting this in. I'm sure it'll be greatly appreciated by those starting a new tokens project.

File: build.gradle (hunk 1 above)
Comment (rusheb) [reply build.gradle:6]: Fixed

File: workflows/build.gradle (hunk 2 above)
Comment (rusheb) [reply workflows/build.gradle]: Fixed

File: build.gradle (hunk 1 above)
Comment (rusheb) [reply build.gradle:6]: What do I need to add to my `build.gradle` to resolve tokens 1.2? If I simply bump the version number I get this error:
```Could not resolve com.r3.corda.lib.tokens:tokens-selection:1.2.

Possible solution:
 - Declare repository providing the artifact, see the documentation at https://docs.gradle.org/current/userguide/declaring_repositories.html

```

I've checked in [Artifactory](https://software.r3.com/artifactory/corda-lib-dev/com/r3/corda/lib/tokens/) and I can't seem to find the JARs

File: workflows/src/test/kotlin/com/template/FlowTests.kt (hunk 3 above)
Comment (rusheb) [reply workflows/src/test/kotlin/com/template/FlowTests.kt]: It looks like `selection` was also moved into `workflows` in `1.2`

https://docs.corda.net/docs/corda-os/4.5/token-sdk-introduction.html#upgrade-from-v-11-to-12

Comment (rusheb) [reply workflows/src/test/kotlin/com/template/FlowTests.kt]: Fixed

File: build.gradle (hunk 1 above)
Comment (rusheb) [reply build.gradle]: Fixed

**Review by rogersanick — APPROVED** [review]
LGTM

**Pull request merged**